src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
@AdamSpeight2008 That is obviously not what @AlekseyTs is suggesting. Your suggestion would break the implementation of `WithLanguageVersion`.

---
In reply to: [74505974](https://github.com/dotnet/roslyn/pull/12969#discussion_r74505974) [](ancestors = 74505974)